src/js/components/modals/ServiceFormModal.js

```diff
@@ -205,13 +206,18 @@ class ServiceFormModal extends mixin(StoreMixin) {
     });
   }
 
+  shouldForceUpdate(message = this.errorMessage) {
```

Comment:
shouldForceUpdate is called in the setStates. This is why this function takes a parameter. But in fact the default is wrong. Changed it.